### ISSUE TITLE
Refs remain unparsed when requesting cross-site

### ISSUE THREAD

**Issue by timkelty** [craftcms/redactor#92]
Example:

`myRedactorField` contains a link to an element on `siteA` (e.g. `<a href=\"{entry:2581:url}\">`)

Twig template `text.twig`: 

```
{% set entry = craft.entries({site: "siteA", id: 123}).one() %}
{{ entry.myRedactorField }}
```

Now request `siteB.com/test.twig`.
The content is returned with the link ref unparsed.

Same issue:
https://github.com/markhuot/craftql/issues/157

**Comment by timkelty** [comment]
It seems that the `siteId` from the entry selection needs to be passed along here, but is not: https://github.com/craftcms/redactor/blob/v2/src/FieldData.php#L47

**Comment by timkelty** [comment]
Related: https://github.com/craftcms/redactor/issues/53

**Comment by brandonkelly** [comment]
This is resolved in Redactor 2.5 thanks to @steverowling’s PR! (2.5 requires Craft 3.4 RC1+)
